integration_test/ci/_config.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-CURRENT_WORKDIR=$PWD
+export CURRENT_WORKDIR=$PWD
 COMPUTATION_ENGINE="../computation-engine-at-orchestrator"
```

Comment:
Had the same comment, but think that could probably be done in a separate issue later.

Reply:
Agree on that it needs to be changed and indeed it is already part of another issue: https://github.com/Project-OMOTES/orchestrator/issues/54 . Would propose to keep it out of this PR to prevent it from growing further in size.